src/migration.rs

```diff
@@ -0,0 +1,111 @@
+//! TODO
+
+use bevy::{
+    ecs::{
+        entity::Entity,
+        reflect::{AppTypeRegistry, ReflectComponent},
+        resource::Resource,
+        system::{Commands, Query, Res, ResMut, RunSystemOnce},
+        world::World,
+    },
+    platform::collections::HashMap,
+    prelude::{Deref, DerefMut},
+    reflect::{FromType, Reflect, prelude::ReflectDefault},
+};
+use std::{
+    any::{Any, TypeId},
+    sync::Arc,
+};
```

Comment:
Should we be using `bevy_platform` here? I don't know how no_std-compatible this crate or subsecond are

Reply:
Moved to core and bevy platform.